app/src/main/java/org/mozilla/fenix/tabtray/TabTrayViewHolder.kt

```diff
@@ -94,49 +98,98 @@ class TabTrayViewHolder(
 
         // Media state
         playPauseButtonView.increaseTapArea(PLAY_PAUSE_BUTTON_EXTRA_DPS)
-        with(playPauseButtonView) {
-            invalidate()
-            Do exhaustive when (store.state.getMediaStateForSession(tab.id)) {
-                MediaState.State.PAUSED -> {
-                    showAndEnable()
-                    contentDescription =
-                        context.getString(R.string.mozac_feature_media_notification_action_play)
-                    setImageDrawable(
-                        AppCompatResources.getDrawable(context, R.drawable.media_state_play)
-                    )
-                }
 
-                MediaState.State.PLAYING -> {
-                    showAndEnable()
-                    contentDescription =
-                        context.getString(R.string.mozac_feature_media_notification_action_pause)
-                    setImageDrawable(
-                        AppCompatResources.getDrawable(context, R.drawable.media_state_pause)
-                    )
+        if (Config.channel.isNightlyOrDebug) {
```

Comment:
It may be easier to introduce a flag in `FeatureFlags` and use that instead:

With that it will be easier to, for example, enable this feature for "beta" without having to change this everywhere. Also it will be easier to find all the code once we can get rid of the flag. :)

Reply:
Sounds good, I'll introduce a feature flag.